tests/unit/test_helpers.js

```diff
@@ -293,4 +299,34 @@ describe(__filename, () => {
       expect(config.get('thisIsAnInvalidKey')).toEqual(value);
     });
   });
+
+  describe('onLocationChanged', () => {
+    // If this test case fails, it means `connected-react-router` has changed
+    // its `onLocationChanged` redux action.
+    // See: https://github.com/mozilla/addons-frontend/issues/6560
+    it('returns the same payload as the connected-react-router action', () => {
+      const history = createMemoryHistory();
+      const { store } = createStore({ history });
+      const dispatchSpy = sinon.spy(store, 'dispatch');
+
+      mount(
+        <Provider store={store}>
+          <ConnectedRouter history={history}>
+            <Route path="/" render={() => <div>Welcome</div>} />
+          </ConnectedRouter>
+        </Provider>,
+      );
+
+      dispatchSpy.resetHistory();
+
+      const pathname = '/foo';
+      history.push(pathname);
+
+      sinon.assert.calledWith(
+        dispatchSpy,
+        onLocationChanged({ pathname, key: sinon.match.string }),
```

Comment:
Ah, yes, I was thinking of `sinon.match.string` but I thought it only worked for mock expectations. I think this will be helpful.

Reply:
Yes, that's super interesting!